src/shapes/textbox.class.js

```diff
@@ -31,6 +31,14 @@
      */
     minWidth: 20,
 
+    /**
+     * Maximum width of textbox, in pixels.
+     * Use the `resizing` event to change this value on the fly
+     * @type {Number | undefined}
+     * @default
+     */
+    maxWidth: undefined,
```

Comment:
i prefer to have Infinity rather than undefined, so we avoid type checking and type complications?

Reply:
I never worked with infinity. If it works... alright.